### PR TITLE
Add import functionality

### PR DESCRIPTION
**Pull request by cmfcmf**
#4

- [x] From the file system (a server directory)
- [x] [Downloads](http://support.zikula.de/module-info-viewpub-tid-1-pid-34.htm)
- [x] [VerySimpleDownload](https://github.com/rallek/VerySimpleDownload)

### CODE REVIEW

File: Resources/views/Import/select.html.twig [hunk 1]
```diff
@@ -3,10 +3,22 @@
 {% block content %}
     <div class="list-group">
         {% for importer in importers %}
-            <a href="{{ path('cmfcmfmediamodule_import_execute', {importer: importer.id}) }}" class="list-group-item">
-                <h4 class="list-group-item-heading">{{ importer.title }}</h4>
-                <p class="list-group-item-text">{{ importer.description }}</p>
-            </a>
+            {% set requirements = importer.checkRequirements %}
+            {% if requirements is same as(true) %}
```
Comment (Guite) [inline Resources/views/Import/select.html.twig]: wouldn't `{% if requirements == true %}` or `{% if requirements is true %}` be enough?

File: Importer/VerySimpleDownloadsModuleImporter.php [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+    public function import($formData, FlashBagInterface $flashBag)
+    {
+        /** @var CollectionEntity $collection */
+        $collection = $formData['collection'];
+        $categoryRegistry = \CategoryRegistryUtil::getRegisteredModuleCategory('CmfcmfMediaModule', 'AbstractMediaEntity', 'Main');
+
+        $conn = $this->em->getConnection();
+        $result = $conn->executeQuery(<<<SQL
+SELECT d.id, d.downloadTitle, d.downloadDescription, d.fileUpload, d.createdUserId, d.updatedUserId, d.createdDate, d.updatedDate, c.categoryId
+FROM vesido_download d
```
Comment (Guite) [inline Importer/VerySimpleDownloadsModuleImporter.php]: `vesido`?

File: Importer/VerySimpleDownloadsModuleImporter.php [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+        return $this->translator->trans('Workflow states are lost and won\'t be imported. All downloads will be visible and approved.', [], 'cmfcmfmediamodule');
+    }
+
+    public function checkRequirements()
+    {
+        $conn = $this->em->getConnection();
+
+        try {
+            $conn->executeQuery('SELECT 1 FROM vesido_download LIMIT 1');
+            $conn->executeQuery('SELECT 1 FROM vesido_download_category LIMIT 1');
```
Comment (Guite) [inline Importer/VerySimpleDownloadsModuleImporter.php]: `vesido`?

File: Resources/views/Import/select.html.twig (hunk 1 above)
Comment (cmfcmf) [inline Resources/views/Import/select.html.twig]: No, because [`true == "foo"`](https://3v4l.org/8mb6T) in which case there is an unfulfilled requirement.

**Pull request merged**